### PR TITLE
KEYCLOAK-7231 KEYCLOAK-7161 Make part of example tests run on app-server-undertow 

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

**Comment by hmlnarik** [comment]
@vramik Could you please review?

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/authorization/DefaultAuthzConfigAdapterTest.java
Comment (vramik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/authorization/DefaultAuthzConfigAdapterTest.java]: Why this is needed?

Comment (mhajas) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/authorization/DefaultAuthzConfigAdapterTest.java]: Because in undertow, probably with jsp (jastow), it doesn't work without / at the end of url.

**Comment by vramik** [comment]
@mhajas ok, thank you. LGTM, after rebase we can merge it.

**Review by vramik — APPROVED** [review]
@mhajas thank you. @hmlnarik as far as I am concerned we can merge this, thanks

**Review by hmlnarik — APPROVED** [review]
Thanks.
Approving based on @vramik's approval

**Pull request merged**
